modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalNavTag.java

```diff
@@ -162,6 +172,9 @@ else if (_active == null) {
 			if (expanded == null) {
 				expanded = Boolean.FALSE;
 			}
+			else {
```

Comment:
here, `expanded` is not `null` but it still can be `false` so we need to check the value